google_deployment_accounts/variables.tf

```diff
@@ -20,6 +20,54 @@ variable "gha_environments" {
   default     = []
 }
 
+# FIXME consider breaking this out into multiple variables
```

Comment:
I think I prefer specifying a single variable to keep the terraform interface simple but it means module users need to build these strings themselves. Using multiple variables instead maps more closely to how we are doing this for github actions, it's just that we happen to only have one variable `input_gha_actions`  on the gha side. I think we did a good job of avoiding the complexity of https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#example-subject-claims by requiring environments in GHA but this isn't really viable for CircleCI.

 I'll likely remove the commit with the commented out version unless somebody has a preference to enumerate each assertion as a variable.